### PR TITLE
style(eltociear): add japanese translated readme

### PR DESCRIPTION
**Pull request by eltociear**
I created Japanese translated README.

### CODE REVIEW

File: README_ja.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+面白い質問やアイデアがあれば、プレイヤーは自分のアイデアを提出することができます。あなたは[プルリクエストの開始](https://github.com/opendilab/LLMRiddles/compare)を行い、私たちに送信することができます。承認後、私たちはそれをレベルに含めます。
+質問形式には以下の点を含めること:
+- プルリクエストタイトル, 例: feature(username): Chapter X-Level Design
+- 記載したい ID
+- 対応する章の問題ファイルを修正する
+- \__init__.py の修正
+
+完全な例については、以下を参照のこと: [自分のレベルデザインを提出する](https://github.com/opendilab/LLMRiddles/pull/6)
+
+## :writing_hand: ロードマップ
```
Comment (TuTuHuss) [inline README_ja.md:75]: Thanks for your contribution, I have a question about this, can this sentence in Japanese also convey the same meaning of future development plans?

Comment (eltociear) [reply README_ja.md:75]: @TuTuHuss Yes!

**Comment by TuTuHuss** [comment]
No more questions, I will merge it.

**Comment by eltociear** [comment]
@TuTuHuss THANKS!

**Pull request merged**